.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
How did this `.idea/.gitignore` file get in? Why not to use global `.gitignore` file with an appropriate comment (that it is related to a particular editor)?